packages/content-api/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@tryghost/content-api",
+  "version": "0.0.0",
+  "repository": "https://github.com/TryGhost/Ghost-SDKs/tree/master/packages/content-api",
+  "author": "Ghost Foundation",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "dev": "echo \"Implement me!\"",
+    "test": "NODE_ENV=testing mocha './test/**/*.test.js'",
+    "lint": "eslint . --ext .js --cache",
```

Comment:
Not necessary for this PR but a future reminder we should add eslint and our eslint plugin unless I'm missing something about how lerna works 😄

Reply:
Deliberately missing because this is a lerna repo :)